src/config.rs

```diff
@@ -67,86 +60,10 @@ impl Default for Server {
     ///
     /// # Panics
     ///
-    /// This function panics if the Server configuration is invalid.
+    /// This function panics if `DB_OFFLINE=leader` but `READ_ONLY_REPLICA_URL` is unset.
```

Comment:
this also looks like an unintentional change, right?

Reply:
Yup, fixed